osu.Game.Rulesets.Catch/Skinning/CatchSkinExtensions.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using JetBrains.Annotations;
+using osu.Framework.Bindables;
+using osu.Game.Rulesets.Catch.UI;
+using osu.Game.Skinning;
+using osuTK.Graphics;
+
+namespace osu.Game.Rulesets.Catch.Skinning
+{
+    internal static class CatchSkinExtensions
+    {
+        [NotNull]
+        public static IBindable<Color4> GetHyperDashFruitColour(this ISkin skin)
```

Comment:
this feels over-engineered and i would just inline, but whatever i guess

Reply:
right actually, things have changed after I dropped changes to the `FruitPiece` component, so this extension may not be of much help now, will inline back.